src/foam/u2/borders/MarginBorder.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.borders',
+  name: 'MarginBorder',
+  extends: 'foam.u2.Element',
+
+  documentation: `
+    A border which adds margin to all sides except the bottom.
```

Comment:
Why not the bottom?

Reply:
oops, that's not true after I found a better way to solve something else